src/Symfony/Component/Console/Tests/Fixtures/Foo1Command.php

```diff
@@ -1,5 +1,14 @@
 <?php
 
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
```

Comment:
Is it php-cs-fixer that caused this license block to be added? That can't be done in this PR because it makes it look like there are a lot more modified files than actually necessary for this feature. 
Please revert all formatting changes not related to the PR.

Reply:
> Is it php-cs-fixer that caused this license block to be added?

Yes, this is due to the php-cs-fixer.

I have used the config .php-cs-fixer.dist.php from the root.
Rule [header_comment](https://github.com/symfony/symfony/blob/9b323c6866078f1bc761d7cc34297c41815108d3/.php-cs-fixer.dist.php#L32) is being configured there.
No idea why the fixtures are also changed.

```
#Executed command:
php tools/php-cs-fixer/vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.dist.php <folder>
```

Reply:
When you pass a directory, you need to use `--path-mode=intersection` (from memory, maybe some typo there) or phpcs ignores the previously defined paths .. and exclusions.

Reply:
Indeed sorry * 

```console
php php-cs-fixer.phar fix --path-mode=intersection /path/to/dir
```
https://cs.symfony.com/doc/usage.html#the-fix-command

(* my local alias for php-cs-fixer is... `phpcs` 😶 )